### ISSUE TITLE
cannot execute "npm install" under example folder

### ISSUE THREAD

**Issue by rainbow494** [xinbenlv/passport-weibo#11]
# issue
- cannot execute "npm install" under the example folder
# the following is exception

```
F:\GitHub\passport\passport-weibo-master\examples\login>npm install
npm WARN Failed to parse json
npm WARN Trailing comma in object at 8:3
npm WARN   }
npm WARN   ^
npm WARN File: F:\GitHub\passport\passport-weibo-master\examples\login\package.json
npm WARN enoent ENOENT: no such file or directory, open 'F:\GitHub\passport\passport-weibo-master\examples\log
npm WARN login No description
npm WARN login No repository field.
npm WARN login No README data
npm WARN login No license field.
```

**Comment by liuyanshi** [comment]
在`/examples/login/package.json`中倒数第三行，也就是`"passport": ">= 0.0.0"`的后面，把逗号`,`去掉就好了

**Comment by xinbenlv** [comment]
Fixed by #8

**Comment by xinbenlv** [comment]
Thank you both for reporting @ly525 and @rainbow494
